resources/js/components/dues/DuesRequiredInfo.vue

```diff
@@ -274,7 +274,10 @@ export default {
       }
 
       return this.selectedPackage.restricted_to_students;
-    }
+    },
+    needsSize: function () {
+      return !this.selectedPackage || (this.merchGroupNames.length > 0) || this.$v.localUser.polo_size.$error || this.$v.localUser.shirt_size.$error;
+    },
```

Comment:
When I tested this locally, the shirt size options correctly disappeared because I'm not a student (although it's odd they appeared initially), but then reappeared when I hit next. I guess you also need to remove the validation requirement for those fields.

Reply:
The visibility logic is based on the condition for the merch dropdown. I figured it's useful to have a shirt size on file for everyone. If it's not already set, I display the option. It'll disappear as soon as you select something, but I'm not super concerned about the edge case. That should only get hit for alumni who never paid dues as a student.

All that said, the implementation itself is glitchy, so I will work that out when I have time.